doc/user/data/sql_funcs.yml

```diff
@@ -376,12 +376,14 @@
     url: extract
 
   - signature: mz_logical_timestamp() -> numeric
-    description: 'The logical time at which a query executes.<br/><br/>**NOTE**: Users
+    description: 'The logical time at which a query executes. Used for temporal filters and internal debugging.<br/><br/>**NOTE**: Users
```

Comment:
The NOTE here isn't accurate. You can use this function in views, as long as its part of a temporal filter.